packages/graphqlgen/src/tests/typescript/__snapshots__/large-schema.test.ts.snap

```diff
@@ -3,7 +3,11 @@
 exports[`large schema tests large schema 1`] = `
 "// Code generated by github.com/prisma/graphqlgen, DO NOT EDIT.
 
-import { GraphQLResolveInfo } from \\"graphql\\";
+import {
+  GraphQLResolveInfo,
```

Comment:
Ideally we wouldn't import these if they are unused, but I'm not too picky about a first version that doesn't have this detail.

Reply:
Actually, was easy to implement, done.